Skip sqlite3 tables with incorrect table_info

I have a third party database that I cannot change it. Is there a way to silently fail and still generate the _correct_ entities?

```
reform-db: 2018/11/10 04:33:51.656895 cmd_init_sqlite3.go:85: no such module: spellfix1
panic: no such module: spellfix1

goroutine 1 [running]:
gopkg.in/reform.v1/internal.(*Logger).Fatalf(0xc0000938f0, 0x45377d0, 0x2, 0xc000348a60, 0x1, 0x1)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/internal/logger.go:51 +0x107
main.initModelsSQLite3(0xc00009c8a0, 0xc0000dbbd0, 0x400b9bd, 0x44dd240)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/cmd_init_sqlite3.go:85 +0x469
main.cmdInit(0xc00009c8a0, 0xc00001e004, 0x1f)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/cmd_init.go:184 +0xb2f
main.main()
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/main.go:128 +0x3a4
```

Something like the following:

https://stackoverflow.com/questions/38425835/reflecting-sqlite-database-with-spellfix1-tables


Edit:
Even better, something like `--skip-tables` I think would be less invasive?

```reform-db: 2018/11/10 04:33:51.656895 cmd_init_sqlite3.go:85: no such module: spellfix1
panic: no such module: spellfix1

goroutine 1 [running]:
gopkg.in/reform.v1/internal.(*Logger).Fatalf(0xc0000938f0, 0x45377d0, 0x2, 0xc000348a60, 0x1, 0x1)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/internal/logger.go:51 +0x107
main.initModelsSQLite3(0xc00009c8a0, 0xc0000dbbd0, 0x400b9bd, 0x44dd240)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/cmd_init_sqlite3.go:85 +0x469
main.cmdInit(0xc00009c8a0, 0xc00001e004, 0x1f)
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/cmd_init.go:184 +0xb2f
main.main()
	/Users/mgonzalez/go/src/gopkg.in/reform.v1/reform-db/main.go:128 +0x3a4```

--- a/reform-db/cmd_init_sqlite3.go
+++ b/reform-db/cmd_init_sqlite3.go
@@ -82,7 +82,9 @@ func initModelsSQLite3(db *reform.DB) (structs []StructData) {
 
 		rows, err := db.Query("PRAGMA table_info(" + tableName + ")") // no placeholders for PRAGMA
 		if err != nil {
-			logger.Fatalf("%s", err)
+			// https://github.com/go-reform/reform/issues/180
+			logger.Printf("Skipping %s: %s.", tableName, err)
+			continue
 		}
 		for {
 			var column sqliteTableInfo

